modelgauge/sut.py

```diff
@@ -61,22 +61,26 @@ class PromptResponseSUT(SUT, ABC, Generic[RequestType, ResponseType]):
 
     @not_implemented
     def translate_text_prompt(self, prompt: TextPrompt) -> RequestType:
+        """Convert the prompt in the the SUT's native representation."""
```

Comment:
Minor typo here and in `translate_chat_prompt:` _Convert the prompt in**to** the ~~the~~ SUT's native representation._

Reply:
Fixed.